gitlab/resource_gitlab_tag_protection.go

```diff
@@ -17,6 +19,10 @@ func resourceGitlabTagProtection() *schema.Resource {
 		Create: resourceGitlabTagProtectionCreate,
 		Read:   resourceGitlabTagProtectionRead,
 		Delete: resourceGitlabTagProtectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabTagProtectionImporter,
```

Comment:
You can use `schema.ImportStatePassthrough` here, since the `Read` function only uses the `id`, not any other attribute.

```suggestion
			State: schema.ImportStatePassthrough,
```